services/worker/src/worker/job_runners/config/parquet_and_info.py

```diff
@@ -18,6 +18,7 @@
 import requests
 from datasets import DownloadConfig, get_dataset_config_info, load_dataset_builder
 from datasets.builder import DatasetBuilder
+from datasets.builder import ManualDownloadError as _ManualDownloadError
```

Comment:
I did it to make it "private" (leading underscore), but I am reverting it.

Reply:
ah ok. What is the benefit of making it private?

Reply:
OK, note that the mypy would give an error anyway because ManualDownloadError is not exported directly by this module.

Something like:

```
src/libcommon/config.py:33: error: Module "worker.job_runners.config.parquet_and_info" does not explicitly export attribute "ManualDownloadError"  [attr-defined]
```